templates/internet-of-things/thank-you.html

```diff
@@ -33,11 +33,11 @@
 var google_conversion_value = 0;
 /* ]]> */
 </script>
-<script type="text/javascript" src="http://www.googleadservices.com/pagead/conversion.js">
+<script type="text/javascript" src="//www.googleadservices.com/pagead/conversion.js">
```

Comment:
Could we explicitly use HTTPS urls? Protocolless URLs are now (arguably) an antipattern. https://twitter.com/paul_irish/status/588502455530311680. Credit to @barrymcgee.